src/p2p/net_node.h

```diff
@@ -459,6 +460,7 @@ namespace nodetool
     epee::math_helper::once_a_time_seconds<60*30, false> m_peerlist_store_interval;
     epee::math_helper::once_a_time_seconds<60> m_gray_peerlist_housekeeping_interval;
     epee::math_helper::once_a_time_seconds<3600, false> m_incoming_connections_interval;
+    epee::math_helper::once_a_time_seconds<95, false> m_seed_pinger;
```

Comment:
95 seconds? Why not a multiple of 30? (Just curious as to rationale)

Reply:
To avoid everything always ticking at the same time.